azure-pipelines.release.yml

```diff
@@ -73,6 +62,13 @@ steps:
       yarn run:published bundle --production
     displayName: yarn bundle
 
+  - script: |
+      npm run publish:beachball -- -b origin/master -n $(npmToken) --access public -y &&
```

Comment:
Same comment about `&&`. Alternately would at least be nice to have a trailing `\` so it is directly transferable to a bash script.